lib/rspec/core/did_you_mean.rb

```diff
@@ -9,17 +10,18 @@ def initialize(relative_file_name)
       end
 
       if defined?(::DidYouMean::SpellChecker)
-        # provide probable suggestions if a LoadError
+        # provide probable suggestions
         def call
           checker = ::DidYouMean::SpellChecker.new(:dictionary => Dir["spec/**/*.rb"])
           probables = checker.correct(relative_file_name)
-          return unless probables.any?
+          return '' unless probables.any?
 
           formats probables
         end
-      else # ruby 2.3.2 or less
-        # return nil if API for ::DidYouMean::SpellChecker not supported
+      else
+        # return a hint if API for ::DidYouMean::SpellChecker not supported
```

Comment:
I think your code is self-explanatory and you don't need the comment. :)

Reply:
I agree, and that has been changed with the refactoring recommended by Jon.